pkg/manager/member/tidb_member_manager.go

```diff
@@ -107,6 +115,13 @@ func (m *tidbMemberManager) Sync(tc *v1alpha1.TidbCluster) error {
 		}
 	}
 
+	isSet, err := m.syncInitializer(tc)
```

Comment:
Check the condition that needs to sync first, if need to sync, call `syncInitializer`, otherwise, continue to sync sts.
No need to return for the error.